plugins/actions/validate.py

```diff
@@ -213,15 +226,6 @@ def validate_rados_options(value):
 osd_options = (
     (optional("dmcrypt"), types.boolean),
     (optional("osd_auto_discovery"), types.boolean),
```

Comment:
Are we still supporting ``osd_auto_discovery`` when ``ceph-disk`` is removed?

Reply:
nop!

Reply:
Actually, this is a regression, can't we use `osd_auto_discovery` with batch or something?

Reply:
There is currently no support for that, batch uses ``devices`` directly. Is ``osd_auto_discovery`` still something we'd like to support?

Reply:
I suspect we have users using it, so I'm afraid to see their experience degraded. Even though there is no support for it anymore...